docs/blockchain-basics/introduction.md

```diff
@@ -15,6 +15,42 @@ import NotificationBar from '../../src/components/docs/NotificationBar';
+"_Chancellor on brink of second bailout for banks_".
+
+At these dates, the financial crisis strikes, it is then easy to reinforce the conclusion that Bitcoin was created mostly for **de-centralisation of money**. To achieve such an aim, Bitcoin uses various Cryptography's techniques and tricks. This explains why we refer to Bitcoin or new systems based on the same mechanics as **crypto-currencies**.
+
+## Terminology: An important note
+With time, the word "cryptocurrency" has been replaced by another more global and mainstream one, because other systems were created to lift Bitcoin's limitations, or propose another phylosophy: _Blockchain_.
+
+The exact word "blockchain" does not appear anywhere in Bitcoin's whitepaper but reflects the technique to arrange and chain datas for the purpose of validation and _immutability_ of exchanges. In Bitcoin's whitepaper, we can find what "blocks" or "chains" are, but they are "just" arranged in a _chain of blocks_.
+
+Hence, the polysemy that the reader would be advised to take into account with caution: "Blockchain" could refer the entire system; or the chained data-structure schema.
+
+## A brief history of the "Blockchain"
+If the newer systems rely directly on Bitcoin or implement their own similar mechanics anyway, Bitcoin itself relies on a lot of pretty old information technologies (IT), from as far as 1973, and until the Genesis Block from 2009. Bitcoin's system is actually around 36 years of research, trials and experiments, and failures.
+
+Needless to say Bitcoin relies on Internet, itself relying on _TCP/IP_ from 1974. But Bitcoin and newer systems also use _Cryptography_, which is, actually, as old as Ancient History. However, what we mean by "Cryptography" in these systems is _Modern Cryptography_, which is linked to first calculating machines and research in modern Mathematics. Very important researchs like for the Diffie, Hellman, and Merkle protocol, or RSA (_Rivest, Shamir, Adleman_) and Public-Key Cryptography, are both respectively from 1976 and 1977.
+
+From there, countless reflexions on ethics, researchs on technologies, and of course Cryptography take place. Mails traceability (1981); Elliptic Curve Cryptography (1985); Digicash (1989); PGP (1991); CyberCash (1994); E-gold (1996); HashCash (1997, with Proof-of-Work); Smart Contracts (1997, defined by Nick Szabo). Even _Bit Gold_ and _B-money_ (from Wei Dai) are from 1998, and the latter already used a decentralized database recording exchanges and using an older version of the Bitcoin's _Proof-of-Work_ (more details in next chapters on this matter).
+
+In 2001, the Bittorent client gave more support for the Peer-2-Peer network trials and researchs, opening the way for more distribution and the decentralisation looked for.
```

Comment:
"opening the way for more distribution and the decentralisation looked for."
incorrect sentencing